apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -218,6 +218,16 @@ fields("cluster") ->
                     desc => ?DESC(cluster_proto_dist)
                 }
             )},
+        {"quic_lb_mode",
+            sc(
+                integer(),
+                #{
+                    mapping => "quicer.lb_mode",
+                    default => 0,
+                    'readOnly' => true,
+                    desc => ?DESC(cluster_quic_lb_mode)
```

Comment:
Since this is not a feature ready for the public yet:

```suggestion
                    mapping => "quicer.lb_mode",
                    importance => ?IMPORTANCE_HIDDEN,
                    default => 0,
                    'readOnly' => true,
                    desc => ?DESC(cluster_quic_lb_mode)
```
?

Reply:
will do it in a follow up when update quicer.